ynput/OpenPype
# fix bug for the update function for 3dsMax loaders

## Changelog Description
Fix the bug of unabling to update/set the version of the published assets in 3dsmax
- [x] maxSceneloader
- [x] fbxloader for camera
- [x] abcloader
- [x] point cloud loader

## Additional Info
MaxScene Loader doesn't support downgrading version
if the version is downgraded, it will show the separate container unparented from the main "_CON" container.

## Testing notes:
1. Launch 3dsMax via Openpype
2. Click Load... in Openpype menu
3.  Load the elder version of the published assets
4. Click Manage... in Openpype menu
5. Click Update to the latest /Switch Version
6. The asset update to the latest version/set version as expected
